src/chapters/booleans-and-conditionals/studio.rst

```diff
@@ -0,0 +1,59 @@
+Studio: Goal Setting and Getting into the Right Mindset
+=======================================================
+
+During this studio, we will ask you to think about your mindset when it comes to facing challenges.
+Do challenges deter you or delight you?
+If challenges are something you delight in, you might have a growth mindset.
+A **growth mindset** can be powered by one word, “yet”.
+The word “yet” allows us to acknowledge that we don’t know something AND that we are capable of learning it.
+For example, with “yet”, the sentence “I don’t understand booleans” becomes “I don’t understand booleans `yet`.”
+
+Growing as developers is something that will take our whole careers.
+No developer knows everything.
+As technology evolves, even the most senior developers have to learn new skills.
+Developing a framework for learning, which includes setting achievable goals, is important for being a developer.
+Technology will change and adapt and we should learn to do the same!
+
+When setting goals, those goals should be SMART. SMART stands for:
+
+**S** pecific
+
+**M** easurable
+
+**A** ttainable
+
+**R** elevant
+
+**T** ime Bound
+
+We will be discussing your goals when it comes to taking this class and your career, but you don't have to do any prior goal setting or preparation to make the most of this activity.
+
+.. note::
+
+   We will be making note of personal goals.
+   If there is a place where you like to write down your personal goals, such as a journal, feel free to bring it to class!
```

Comment:
Good tip, but the way it is phrased contradicts the "you don't have to do any prior goal setting" comment above.

Reply:
Edited it! Thanks!